poe-api/src/poe.py

```diff
@@ -147,6 +147,10 @@ def get_bot_names(self):
       bot_names[bot_nickname] = bot_obj["displayName"]
     return bot_names
   
+  def get_remaining_messages(self, chatbot):
+    chat_data = self.get_bot(chatbot)
```

Comment:
`client.get_bot` takes the bot display name as an input, so you'd have to do `self.get_bot(self.bot_names[chatbot])`

Reply:
I used to use the name "GPT-4" directly but it's true that using the official code names is better